web/pgadmin/static/js/SchemaView/index.jsx

```diff
@@ -784,39 +798,39 @@ function SchemaDialogView({
 
   /* I am Groot */
   return (
-    <StateUtilsContext.Provider value={stateUtils}>
-      <DepListenerContext.Provider value={depListenerObj.current}>
-        <Box className={classes.root}>
-          <Box className={classes.form}>
+    <StyledBox className='Dialog-root'>
```

Comment:
no need to have class on root if it is not conditional

Reply:
Done